modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/model/ContactOption.kt

```diff
@@ -0,0 +1,21 @@
+package edu.stanford.spezi.modules.contact.model
+
+import androidx.compose.ui.graphics.vector.ImageVector
+import java.util.UUID
+
+/**
+ * ContactOption data class used to represent a contact option.
+ *
+ * @param id the unique identifier of the contact option
+ * @param image the image of the contact option
+ * @param title the title of the contact option
+ * @param action the action of the contact option
+ */
+data class ContactOption(
+    val id: UUID = UUID.randomUUID(),
+    val image: ImageVector?,
+    val title: String,
```

Comment:
Following the image comment, we have already introduced a similar type to handle strings used in compose, have a look at `StringResource` and it's text method usage in compose